src/inverseRobot.js

```diff
@@ -24,7 +24,15 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
-}
+  const newRobot = {};
+
+  for (const key in robot) {
+    if ((newRobot.hasOwnProperty(robot[key]))) {
```

Comment:
```suggestion
    if (newRobot.hasOwnProperty(robot[key])) {
```
Зачем тут дополнительные скобки?